src/Media_Command.php

```diff
@@ -360,6 +360,82 @@ function import( $args, $assoc_args = array() ) {
 		}
 	}
 
+	/**
+	 * List media sizes registered with WordPress.
+	 *
+	 * ## OPTIONS
+	 *
+	 * [--fields=<fields>]
+	 * : Limit the output to specific fields. Defaults to all fields.
```

Comment:
Although it becomes clear as soon as you run it, it'd be nice to have a list of the fields here.